app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/pref/Prefs.kt

```diff
@@ -18,4 +19,21 @@ object Prefs : KotprefModel() {
             context.bool(R.bool.pref_default_value_enable_hide_bottom_navigation),
             R.string.pref_key_enable_hide_bottom_navigation
     )
+    var enableReopenPreviousRoomSessions: Boolean by booleanPref(
+            context.bool(R.bool.pref_default_value_enable_reopen_previous_room_sessions),
+            R.string.pref_key_enable_reopen_previous_room_sessions
+    )
+    var previousSessionTab: String by stringPref(
```

Comment:
Can you use the enum support? 👀
"com.chibatching.kotpref:enum-support:2.3.0"
https://github.com/chibatching/Kotpref#declare-preference-model

Reply:
Let' use this!
Thanks, and sorry for my shortage of investigation

Reply:
It's ok 🙆 
I study a lot by your code 🙇